examples/datasets/plot_limo_data.py

```diff
@@ -33,146 +35,282 @@
+# ``limo_epochs.metadata``: phase-coherence and Face A vs. Face B.
+
+# name of predictors + intercept
+predictor_vars = ['face a - face b', 'phase-coherence', 'intercept']
+
+# create design matrix
+design = limo_epochs.metadata[['phase-coherence', 'face']].copy()
 design['face a - face b'] = np.where(design['face'] == 'A', 1, -1)
-names = ['intercept', 'face a - face b', 'phase-coherence']
+design['intercept'] = 1
+design = design[predictor_vars]
+
+###############################################################################
+# Now we can set up the linear model to be used in the analysis using
+# MNE's linear_regression function (see func:`mne.stats.linear_regression`).
 
-# fit linear model
-reg = linear_regression(limo_epochs, design[names], names=names)
+reg = linear_regression(limo_epochs,
+                        design_matrix=design,
+                        names=predictor_vars)
 
 ###############################################################################
-# Visualise effect of phase-coherence.
+# Extract regression coefficients
+# -------------------------------
+#
+# The results stored within the object ``reg``.
+# It basically consists of a dictionary of evoked objects containing
+# multiple inferential measures for each predictor in the design matrix.
+
+print('predictors are:', [key for key in reg.keys()])
+print('fields are:', [field for field in getattr(reg['intercept'], '_fields')])
+
+###############################################################################
+# Plot model results
+# ------------------
+#
+# Now we can access and plot the results of the linear regression analysis by
+# calling `reg['<name of predictor>'].<measure of interest>` and using the
+# `.plot_joint()` method just as we would do with any other evoked object.
+# Below we can see a clear effect of phase-coherence, with higher
+# phase-coherence (i.e., better "face visibility") having a negative effect on
+# the activity measured at occipital electrodes around 200 to 250 ms following
+# stimulus onset.
+
 reg['phase-coherence'].beta.plot_joint(ts_args=ts_args,
-                                       title='Effect of phase-coherence',
-                                       times=[.23])
+                                       title='Effect of Phase-coherence',
+                                       times=[0.23])
 
 ###############################################################################
-# Here we can see a clear effect of phase-coherence, with higher
-# phase-coherence (i.e., better "face visibility") being associated with
-# stronger activity patterns.
+# We can also plot the corresponding T values.
+
+# use unit=False and scale=1 to avoid conversion keep values at their original
+# scale (i.e., avoid conversion to micro-volt).
+ts_args = dict(xlim=(-0.25, 0.5),
+               unit=False)
+topomap_args = dict(scalings=dict(eeg=1),
+                    average=0.05)
+
```

Comment:
```suggestion

# sphinx_gallery_thumbnail_number = 9
```

Reply:
this will determine which figure from the tutorial gets used on the examples gallery page.